docs/user/ComplianceChecking.md

```diff
@@ -0,0 +1,121 @@
+
+**Note** the Namespace is used to identify all Topology Templates that are subject to the rule. 
+This means each Compliance Rule defined for a certain namespace automatically applies to all Topology Templates defined in that namespace.
+
+**Step.2.1** Model Identifier and Required Structure
+
+After the previous steps a compliance rule with empty Identifier and Required Structure is created. 
+The Topology Modeler of Winery allows to model the identifier and required structure of the compliance rule:  
+![step2.PNG](graphics/compliance-checking/step2.PNG)
+1. Choose Tab "Identifier" or "Required Structure"
+2. click "open editor"
+3. model "Identifier" or "Required Structure"
+
+An exemplary compliance rule that is concerned with the storage of personal user data may look like this:
+
+Identifier             |  Required Structure
+:-------------------------:|:-------------------------:
+![identifier_modeler.PNG](graphics/compliance-checking/identifier_modeler.PNG)  |  ![required-structure.PNG](graphics/compliance-checking/required-structure.PNG)
+
+The motivation for this compliance rule is to find all MySQL databases that store personal data and to ensure that it is hosted on a specific OpenStack Component with HypervisorEndpoint set to "192.168.4.3" 
+
+
+
+**Step 3:** Check Compliance of a Topology Template
+
+After creating our first compliance rule we can check if a Topology Template with the same namespace satisfies this rule.
+Therefor, we can either use an existing Service Template or create a new Service Template.
+Note that it is important to ensure that the compliance rule is detected in your Topology Template.
+
+1. Click on the "Service Templates" tab
+2. Select a Service Template that you want to check
+3. Click on the "Constraint Checking" tab
+4. Click on the button "Check Constraints"
+5. The results will be shown as XML
+
+
+An example of a Topology Template that does not satisfy the compliance rule defined in Step 2.1 is shown below:
+
+Topology Template             |  Result
+:-------------------------:|:-------------------------:
+![unsatisfied-topology.PNG](graphics/compliance-checking/unsatisfied-topology.PNG)  |  ![unsatisfied.PNG](graphics/compliance-checking/unsatisfied.PNG)
+
+The left side shows the modeled Topology Template.
+The specified property HypervisorEndpoint: "192.168.0.0" of the OpenStack component violates constraints specified by the compliance rule.
+The result indicates the compliance rule that is violated.
+
+After updating the property HypervisorEndpoint to "192.168.4.3" the Topology Template does satisfy the compliance rule defined in Step 2.1 as shown below:
+
+Topology Template             |  Result
+:-------------------------:|:-------------------------:
+![satisfied-topology.PNG](graphics/compliance-checking/required-structure.PNG)  |  ![satisfied.PNG](graphics/compliance-checking/satisfied.PNG)
+
+
+ 
+
+
+
+
+## License
+
+Copyright (c) 2013-2018 Contributors to the Eclipse Foundation
```

Comment:
This file was created recently: the year is only 2018.

Reply:
thanks for the comments i will fix them :)